Library/Formula/pjproject.rb

```diff
@@ -0,0 +1,19 @@
+class Pjproject < Formula
+  homepage "http://www.pjsip.org/"
+  url "http://www.pjsip.org/release/2.3/pjproject-2.3.tar.bz2"
+  sha256 "e7fa60a3b59424430145af90372282ca778449f7b68b77bb24a9cf75d94d5765"
+
+  depends_on "openssl"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "dep"
+    system "make"
+    system "make", "install"
+    bin.install "pjsip-apps/bin/pjsua-#{`uname -m`.chomp}-apple-darwin#{`uname -r`.chomp}" => "pjsua"
+  end
+
+  test do
+    assert_match /PJ_VERSION\s*: 2.3/, shell_output("#{bin}/pjsua --version")
```

Comment:
It's a SIP user agent (client), similar to an IM client. How would you test a command-line IRC/XMPP/email client? I can't rely on an ability to connect to any particular server as a test. What would be satisfactory?